httpapi/coreapi/types.go

```diff
@@ -58,13 +67,13 @@ type ResponseHeader struct {
 
 // DocumentResponse is the common response for Document APIs.
 type DocumentResponse struct {
-	Header     ResponseHeader `json:"header"`
-	Scheme     string         `json:"scheme" enums:"generic,invoice,purchase_order,entity"`
-	Data       interface{}    `json:"data"`
-	Attributes AttributeMap   `json:"attributes"`
+	Header     ResponseHeader       `json:"header"`
+	Scheme     string               `json:"scheme" enums:"generic,invoice,purchase_order,entity"`
```

Comment:
Do you think it is possible to get tag from a config file. `json:"scheme" enums:"generic,invoice,purchase_order,entity"`

Let's say a new dev is trying to add a new doc type.
I guess it is a bit hard to figure out you have to change some tags.